components/UI/PortfolioItem.jsx

```diff
@@ -30,8 +30,8 @@ const PortfolioItem = (props) => {
           )}
 
           <div className='bg-transparent'>
-            <div className={`${classes.portfolio__img}`}>
-              <Image alt={title} src={img} width={380} height={1} />
+            <div className={`${classes.portfolio__img}`} >
+              <Image alt={title} src={img} width={380} height={1} style={{maxHeight: "380px"}}/>
```

Comment:
Maybe we require some kind of overflow statement here as we are fixing the maxHeight 🙌🏻

Reply:
sorry I didnt understand, could you please elaborate what is expected

Reply:
@Sameerdeshmukh18 ,if the image's height exceeds the specified maximum height of 380 pixels, then the [overflow](https://www.w3schools.com/css/css_overflow.asp) property comes into play. Example :  
` style={{ maxHeight: "380px", overflow: "auto" }}`

Reply:
yeah ok
I will do that